resources/views/content/scripts.twig

```diff
@@ -7,10 +7,9 @@
             action = '/' + App.language + '/search';
         }
 
-        formElement.setAttribute('action', action);
-        formElement.submit();
+        query = document.querySelector("input.search-input").value;
```

Comment:
This variable was not declared before, but since this is JavaScript, it allows it anyway. Please declare it using `var`.